src/org/omegat/Bundle.properties

```diff
@@ -1803,6 +1805,7 @@ WG_SAVE_ORIGIN=Save origin of &translation
 WG_INITIAL_SEGMENT_LOAD_COUNT=Initial number of loaded segments:
 WG_INITIAL_SEGMENT_LOAD_COUNT_TOOLTIP=When a single file has more segments than this, additional segments will be loaded on-demand as you scroll.
 WG_PARA_MARK=Paragraph delimiter:
+WG_SUPPORT_LOCKED_SEGMENTS=Support for locked segments
```

Comment:
I think "Allow segment locking" would be better for this one, if I understand the intent correctly.